src/main/java/com/easyrest/pages/AdministratorPage.java

```diff
@@ -0,0 +1,106 @@
+package com.easyrest.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class AdministratorPage extends BasePage {
+
+    private final By logo = By.xpath("//a[text()='Easy-rest']");
+    private final By home = By.xpath("//span[contains(.,'Home')]");
+    private final By restaurantList = By.xpath("//span[contains(.,'Restaurants List')]");
+    private final By userMenu = By.xpath("//div[contains(@class,'UserMenu-root')]//button");
+
+    private final By administratorPanel = By.xpath("//a[@role='menuitem']");
+    private final By logOut = By.xpath("//li[@role='menuitem']");
```

Comment:
This class can be without locators to header and UserRoleMenuPanel because they are in component

Reply:
I totally agree with you.  I will replace it when UserRoleMenuPanel component is in main